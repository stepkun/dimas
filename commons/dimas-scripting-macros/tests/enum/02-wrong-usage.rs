// Copyright Â© 2025 Stephan Kunz

//! Test wrong usage of scripting enum derive macro `ScriptEnum` 

#[dimas_scripting_macros::ScriptEnum]
enum TestEnum {
    CaseA,
    CaseB,
}

// dummy main
fn main(){}
