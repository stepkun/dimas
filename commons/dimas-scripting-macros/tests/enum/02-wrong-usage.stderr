error: expected attribute, found derive macro `dimas_scripting_macros::ScriptEnum`
 --> tests/enum/02-wrong-usage.rs:5:3
  |
5 | #[dimas_scripting_macros::ScriptEnum]
  |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not an attribute
